zerver/migrations/0559_stream_is_recently_active.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 5.0.6 on 2024-07-29 09:27
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("zerver", "0558_realmuserdefault_web_animate_image_previews_and_more"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="stream",
+            name="is_recently_active",
+            field=models.BooleanField(default=True),
```

Comment:
Do we want to set `db_default=True` too? That would make it safer to migrate to this PR.

Reply:
Done